src/wallet.ts

```diff
@@ -78,6 +78,17 @@ export default class Wallet {
     return newAccount.address;
   }
 
+  /**
+   * Adds a new account from the given private key and sets the default account as a new account.
```

Comment:
s/sets the default account as a new account/sets the new account as the default account/?

Reply:
Thanks! I'll change it!